programs/README.md

```diff
@@ -0,0 +1,30 @@
+# Foundation Program RFCs
+
+There are many things that this new foundation can do to support its projects! To capture and flesh out as may of these as possible, we will use an RFC process to accept and evaluate the proposals in this repo. Some of these program proposals may be determined to be required for Minimum Viable Foundation. These proposals
+
+## When you need to follow this process
+
+You need to follow this process if you intend to introduce a new program proposal to this repo. If you would like to offer changes to an existing proposal or governance document, please simply PR against the existing file.
+
+If you submit a pull request to add a new document without going through the RFC process, it may be closed with a polite request to submit an RFC first!
+
+## To begin drafting a program proposal:
+
+ 1. Fork the [nodejs/bootstrap repo](https://github.com/nodejs/bootstrap).
+ 2. Copy `programs/0000-template.md` to `programs/0000-my-program-name.md` (where 'my-program-name' is descriptive. Don't assign the RFC a number yet).
+ 3. Fill in the RFC template. Put care into the details!
+ 4. Submit a pull request. As a pull request, the draft will receive feedback from the larger community, and the author should be prepared to revise it in response.
+ 5. Build consensus and integrate feedback.
+ 6. Eventually, the Bootstrap co-chairs will decide in one of their weekly meetings whether the RFC is a candidate for inclusion.
+
+RFCs that are candidates for inclusion will enter a "final comment period" lasting 7 days. The beginning of this period will be signaled with a comment and tag on the RFC's pull request. The RFC can be modified based upon feedback from the community. Significant modifications may trigger a new final comment period. At the close of this review period, the RFC may be:
+
+ * accepted at the close of its final comment period. A Bootstrap member will merge the RFC's associated pull request, at which point the RFC will become 'active', or;
+
+ * rejected by the Website Redesign working group after public discussion has settled and comments have been made summarizing the rationale for rejection. A member of the working group should then close the RFC's associated pull request.
```

Comment:
I think this is cut and paste error here.   `the Website Redesign working group`  should be removed.  

`A member of the working group` should be `A Bootstrap member`